app/src/views/TermsAndConditions/i18n.json

```diff
@@ -0,0 +1,17 @@
+{
+    "namespace": "termsAndConditionsSS",
```

Comment:
Let's make sure to always verify our PRs ourselves first, as this seems to be a typo.